kind/actions/kind.go

```diff
@@ -165,7 +165,7 @@ func DeleteControlPlane(clusterName, nodeName string) error {
 
 	// pick the first one, but there should never be multiples since docker requires name to be unique.
 	node := nodeList[0]
-	if err := DeleteClusterNode(clusterName, nodeName); err != nil {
+	if err := DeleteNode(clusterName, nodeName); err != nil {
```

Comment:
This change is unrelated to the purpose of the PR. Is intentional to add it here?